internal/host/hostcommands/download_boot_artifacts_cmd.go

```diff
@@ -46,6 +46,11 @@ func (c *downloadBootArtifactsCmd) GetSteps(ctx context.Context, host *models.Ho
 	if err != nil {
 		return nil, err
 	}
+	// Only validate OS image if image service is enabled
+	if osImages := c.osImages.GetOpenshiftVersions(); len(osImages) == 0 {
```

Comment:
It's a bit strange that we use an empty os image list here to mean the image service is disabled, but we have the flag explicitly everywhere else.

Is there a reason we can't use the flag here too?